apps/dashboard/src/components/subscribers/subscriber-row.tsx

```diff
@@ -0,0 +1,150 @@
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuGroup,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from '@/components/primitives/dropdown-menu';
+import { Skeleton } from '@/components/primitives/skeleton';
+import { TableCell, TableRow } from '@/components/primitives/table';
+import { TimeDisplayHoverCard } from '@/components/time-display-hover-card';
+import TruncatedText from '@/components/truncated-text';
+import { useEnvironment } from '@/context/environment/hooks';
+import { formatDate } from '@/utils/format-date';
+import { buildRoute, ROUTES } from '@/utils/routes';
+import { cn } from '@/utils/ui';
+import { ISubscriberResponseDto } from '@novu/shared';
+import { ComponentProps } from 'react';
+import { RiFileCopyLine, RiMore2Fill, RiPulseFill } from 'react-icons/ri';
+import { Link } from 'react-router-dom';
+import { Avatar, AvatarFallback, AvatarImage } from '../primitives/avatar';
+import { CompactButton } from '../primitives/button-compact';
+import { CopyButton } from '../primitives/copy-button';
+
+type SubscriberRowProps = {
+  subscriber: ISubscriberResponseDto;
+};
+
+type SubscriberLinkTableCellProps = ComponentProps<typeof TableCell> & {
+  subscriber: ISubscriberResponseDto;
+};
+
+const SubscriberLinkTableCell = (props: SubscriberLinkTableCellProps) => {
+  const { subscriber, children, className, ...rest } = props;
+
+  return (
+    <TableCell className={cn('group-hover:bg-neutral-alpha-50 text-foreground-600 relative', className)} {...rest}>
+      {children}
+    </TableCell>
+  );
+};
+
+export const SubscriberRow = ({ subscriber }: SubscriberRowProps) => {
+  const { currentEnvironment } = useEnvironment();
+
+  return (
+    <TableRow key={subscriber.subscriberId} className="group relative isolate">
+      <SubscriberLinkTableCell subscriber={subscriber}>
+        <div className="flex items-center gap-3">
+          <Avatar>
+            <AvatarImage src={subscriber.avatar || undefined} />
+            <AvatarFallback className="bg-neutral-alpha-100">
+              {subscriber.firstName?.[0] || subscriber.email?.[0] || subscriber.subscriberId[0]}
+            </AvatarFallback>
+          </Avatar>
+          <div className="flex flex-col">
+            <TruncatedText className="text-foreground-950 max-w-[32ch] font-medium">
+              {subscriber.firstName || subscriber.email || subscriber.subscriberId}
```

Comment:
the text doesn't truncate well
![Screenshot 2025-01-21 at 15 17 05](https://github.com/user-attachments/assets/446b7908-023c-4f3a-8854-5e4fca59d017)

Reply:
This works as expected in my local env.
<img width="424" alt="Screenshot 2025-01-21 at 4 35 19 PM" src="https://github.com/user-attachments/assets/fbddb83d-e0de-4983-aace-2f9edc588516" />

Reply:
ok, maybe you pushed some changes, I'll refresh

Reply:
ok fixed